src/status/status-module.ts

```diff
@@ -241,12 +241,11 @@ export default class StatusModule {
   }
 
   public async getStatus(withEcs = false) {
-    const [gitlab, runners, postgresql, screenshotter, systemHook, charlesVersion] = await Promise.all([
+    const [gitlab, runners, postgresql, screenshotter, charlesVersion] = await Promise.all([
       this.getGitlabStatus(),
       this.getRunnersStatus(),
       this.getPostgreSqlStatus(),
       this.getScreenshotterStatus(),
-      this.getSystemHookStatus(),
```

Comment:
Why was this dropped? And if it's not needed, should we remove the method as well?